aea/connections/scaffold/readme.md

```diff
@@ -0,0 +1,5 @@
+# Scaffold connection
+The scaffold connection acts as a boilerplate for a newly created connection.
+
+## Usage
+Create a scaffold connection with the `aea scaffold connection {NAME}` command and implement your own connection.
```

Comment:
Minor: maybe use `${NAME}`? That would be `sh` syntax.

Or, just use `NAME` because that's the standard way to document positional and mandatory arguments in a CLI command.